### PR TITLE
Iframes4all

### PR DESCRIPTION
**Pull request by spennythug**

### CODE REVIEW

**Comment by mfairchild365** [comment]
Why two pull requests for the same thing?

**Comment by spennythug** [comment]
I forgot I opened the other one.

**Comment by mfairchild365** [comment]
You should close one of them.

**Comment by spennythug** [comment]
make me

File: www/templates/html/Media.tpl.php [hunk 1]
```diff
@@ -135,7 +130,17 @@
...
+                
+                <?php if($getTracks){
+                    echo '<div class="mediahub-onpage-captions">';
+                    echo $savvy->render($context, 'MediaPlayer/Transcript.tpl.php');
+                    echo '</div>';
+                    }
+                ?>
+               
+                <hr>
+                <h6 class="wdn-sans-serif">Description</h6>
```
Comment (mfairchild365) [inline www/templates/html/Media.tpl.php]: is this really an h6?

File: www/templates/html/Media.tpl.php [hunk 2]
```diff
@@ -164,7 +169,7 @@
                     <script type="text/javascript">
                         WDN.loadCSS('../templates/html/css/comments.css?v=<?php echo UNL_MediaHub_Controller::VERSION ?>');
                     </script>
-                    <h6 class="wdn-sans-serif">COMMENTS <span class="wdn-icon wdn-icon-comment"></span></h6>
+                    <h6 class="wdn-sans-serif">Comments <span class="wdn-icon wdn-icon-comment"></span></h6>
```
Comment (mfairchild365) [inline www/templates/html/Media.tpl.php]: is this really an h6?

File: www/templates/html/MediaPlayer/Transcript.tpl.php [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                        <li>Click any line to jump to that spot in the video. </li>
+                        <li>Use the icons to the right to toggle between list and paragraph view. </li>
+                    </ul>
+                </div>
+            </div>
+            <button class="mh-caption-search-close caption-toggle" aria-label="Close Searchable Transcript">x</button>
+            <select id="mh-language-select" aria-label="select language for searchable transcript">
+                <?php 
+                    $languageTracks = array_keys($tracks);
+                    $i = 0;
```
Comment (mfairchild365) [inline www/templates/html/MediaPlayer/Transcript.tpl.php]: Is this actually being used?

File: www/templates/html/Media.tpl.php (hunk 1 above)
Comment (spennythug) [inline www/templates/html/Media.tpl.php]: yup! see how it says "h6"?

File: www/templates/html/Media.tpl.php (hunk 2 above)
Comment (spennythug) [inline www/templates/html/Media.tpl.php]: I know it's hard to believe but it's really there!

File: www/templates/html/MediaPlayer/Transcript.tpl.php (hunk 3 above)
Comment (spennythug) [inline www/templates/html/MediaPlayer/Transcript.tpl.php]: Depends on who you ask

**Comment by mfairchild365** [comment]
@spennythug can you close this one and open a new so that that it will be merged into the `iframe-embeds` branch?

**Comment by spennythug** [comment]
@mfairchild365 I will do no such thing.

**Comment by spennythug** [comment]
@mfairchild365 If you'd prefer I can force push a gif of a dancing pickle to that branch.

**Pull request closed without merging**